app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MovieActorsController < ApplicationController 
+  def create
+    @individual_movie = Movie.find(params[:id])
+    new_actor = @individual_movie.actors.new(actor_params)
+    new_actor.save
```

Comment:
Nice job with this set up! I agree with doing this work in the `#create` method. I think you can make this even slimmer by just doing `@movie.actors.create(actor_params)` instead of separating out into `.new` and .`save` - I'm guessing you did this in case of implementing some sad path messaging, which is totally ok too.